tests/test_download.py

```diff
@@ -69,6 +69,7 @@ def download_request_fixture(output_folder):
     )
 
 
+@pytest.mark.sh_integration
```

Comment:
In this module I wouldn't mark any of the tests with `sh_integration`. This roda.sentinel-hub.com is actually just a http access to info about the content of Sentinel S3 buckets. It is completely public.